javascript/index.js

```diff
@@ -0,0 +1,8 @@
+import { HttpClient } from '@essential-projects/http';
```

Comment:
Please keep to the commonly used style.

```suggestion
import {HttpClient} from '@essential-projects/http';
```